tools/integration_tests/util/setup/implicit_and_explicit_dir_setup/implicit_and_explicit_dir_setup.go

```diff
@@ -72,15 +76,61 @@ func RemoveAndCheckIfDirIsDeleted(dirPath string, dirName string, t *testing.T)
 	}
 }
 
-func CreateImplicitDirectoryStructure(testDir string) {
+type objectCreationMetadata struct{ content, completeObjectName string }
+
+func createObjectsOnGcs(objects []objectCreationMetadata, t *testing.T) {
+	ctx := context.Background()
```

Comment:
We have created [storageClient for package](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/implicit_dir/implicit_dir_test.go#L50), please use that instead of creating new one.

Reply:
Done

Reply:
Never mind. I had made the change for this fix. But it somehow got lost in rebase. I'll find and add it back. Keeping this comment open till then..

Reply:
found and added back the lost commit.